src/utils/helpers.js

```diff
@@ -0,0 +1,5 @@
+export const titleCase = str =>
+  str
+    .split(' ')
+    .map(word => word.replace(word[0], word[0].toUpperCase()))
```

Comment:
bonus points if you can do this with a regex. 😈

Reply:
fixed with CSS in a previous commit